engine/src/main/java/org/terasology/physics/engine/RigidBody.java

```diff
@@ -120,6 +121,9 @@
      */
     void setOrientation(Quat4f orientation);
 
+    Matrix4f getWorldTransform();
+    Matrix4f setWorldTransform(Matrix4f trans);
```

Comment:
Will it be better to use the `Transform` class here (or modify `Transform` to be created from/use `Matrix4f`) instead?